src/inverseRobot.js

```diff
@@ -23,8 +23,53 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  if (!robot) {
+    return {};
+  }
```

Comment:
`@param {object} robot`

I guess it's a bit of an overkill, but anyway, GJ

Reply:
now I know about params)
deleted this